src/index.js

```diff
@@ -1,20 +1,29 @@
-import * as calendarUtils from "./utils/CalendarUtils/CalendarUtils";
-import * as proxy from "./utils/Proxy/Proxy";
 import * as main from "./Main/Main";
+import defaultProxyServerUrl from "./resources/DefaultProxyServerUrl/DefaultProxyServerUrl";
+import { isDefined } from "./utils/JavaScriptUtils/JavaScriptUtils";
 
-const index = (
+export default (
   container,
-  gitHubUsers = [],
-  gitLabUsers = [],
-  proxyServerUrl = proxy.defaultProxyServerUrl
+  gitHubUsers,
+  gitLabUsers,
```

Comment:
This is a breaking change in the api! We should never casually do this.

> removed default values set to the user arrays(it was confusing because it didn't always default to the empty array in case of certain values, e.g. null and "")

Why should it? Explicitly passing an argument will make it so that it doesn't default.

Overall I don't see what this changes except that user must provide e.g. `gitLabUsers` that I'm sure many would skip.

Reply:
> Why should it? Explicitly passing an argument will make it so that it doesn't default.

Explicitly passing `undefined` makes it so that it defaults to the empty arrays. I'd expect it so in the case of falsy values as well.

Reply:
> Overall I don't see what this changes except that user must provide e.g. gitLabUsers that I'm sure many would skip.

In its current form, passing a falsy value(e.g. `null` and `""`) to either of the user arrays would break the flow. In these cases the arguments do *not* default to the empty arrays. What I'd like to make sure is that the user-defined arguments are in the expected/valid format. Doesn't it make sense?

Reply:
Those are invalid inputs so they should break the flow, shouldn't they?

Reply:
> In its current form, passing a falsy value(e.g. null and "")

You mean this? Yep, they'll because they're not iterable.

Reply:
You added a check for the value not being an array, so it either way it would fail.

Reply:
Gotcha. I guess then there's nothing to do with the changes.

Reply:
This is a breaking change in the api which we shouldnt introduce and also removes some use cases so the defaults should be re-added.

Reply:
This is a breaking change in the api which we shouldnt introduce and also removes some use cases so the defaults should be re-added.

Reply:
Re-added default arguments in b9bea9c.